bai1/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace bai1
```

Comment:
```suggestion
namespace Bai1
```
tên namespace thì cần viết theo kiểu CamelCase nhé

Reply:
vâng anh..